final_prep/annotations.rb

```diff
@@ -3,41 +3,69 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# A bear is being defined by its attributes
 def build_a_bear(name, age, fur, clothes, special_power)
+  # The variable, greeting, is being defined as a string. A name can be inserted in the string.
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # The variable, demographics is being associated to the attributes, "name" and "age" in an array.
   demographics = [name, age]
+  # The variable, power_saying, is being defined in a string. A special power can be inserted in the string.
   power_saying = "Did you know that I can #{special_power}?"
+  # A hash is being created. The keys and values are being defined.
   built_bear = {
+    # the key, "basic_info", is being associated to the "Demographics" variable.
     'basic_info' => demographics,
+    # The key, "clothes", is being associated to the "clothes" attribute.
     'clothes' => clothes,
+    # The key, "exterior", is being associated to the "fur" attribute.
```

Comment:
`Key` and `attribute` are synonymous here. Another way to think about it is that the `key` or `attribute` "exterior" is being assigned a value of "fur".